icicle/appUtils/vector_manipulation/ve_mod_mult.cu

```diff
@@ -7,92 +7,6 @@
 #include "ve_mod_mult.cuh"
 
 
-#define MAX_THREADS_PER_BLOCK 256
```

Comment:
Changed to only prototypes in lde.cuh but not in ve_mod_mult.cuh because if I try, a linking error appears. It basically says that functions from ve_mod_mult.cuh can't be found. I'm not a C++ expert but it seems like this can be relevant: https://stackoverflow.com/questions/495021/why-can-templates-only-be-implemented-in-the-header-file